stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: grafana
-version: 1.14.6
-appVersion: 5.2.4
+version: 1.14.7
+appVersion: 5.2.5
```

Comment:
Why is the `appVersion` changing? The default `image.tag` is still [5.2.4](https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L14).